types/gl-matrix/index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Type definitions for gl-matrix 2.3.2
+// Type definitions for gl-matrix 2.4.0
```

Comment:
Could you enable linting to in `tslint.json` to correctly parse the header? It should only be versioned at 2.4 (major.minor, not patch).

Reply:
Fixed. Thanks for pointing that out!